debian/control

```diff
@@ -29,7 +29,7 @@ Build-Depends: debhelper (>= 10),
                libboost-dev,
                libspdk-dev,
                libipmctl-dev,
-               libraft-dev,
+               libraft-dev (= 0.7.0-1377.g4faff0b),
```

Comment:
does this require such a specific value rather than 0.7.0?

Reply:
When it's an `=`, I believe it does.  If you look at c9b6493f42d1ec921aaac03787dc1a21d3cfb789 you can see @liw tried https://github.com/daos-stack/daos/blob/f6431fac9f31d8c38ebab6fbcac5322fcc136f2a/debian/control#L32 and in the [DEB Build](https://build.hpdd.intel.com/blue/rest/organizations/jenkins/pipelines/daos-stack/pipelines/daos/branches/PR-3826/runs/4/nodes/99/steps/310/log/?start=0) you can see that it couldn't find `libraft-dev (= 0.7.0)`.

If the `Build-Depends` were changed to `libraft-dev (>= 0.7.0)` that would probably work.
`